src/connection-manager/update-amendments-from-json.ts

```diff
@@ -0,0 +1,63 @@
+import 'dotenv/config'
+import * as fs from 'fs'
+
+import { rippleTimeToUnixTime } from 'xrpl'
+
+import { query } from '../shared/database/utils'
+import { AmendmentEnabled } from '../shared/types'
+import logger from '../shared/utils/logger'
+
+const log = logger({ name: 'database-agreement' })
+const filePath = 'src/shared/data/amendments_enabled.json'
+
+interface AmendmentEnabledJson {
+  networks: string
+  amendments: Array<{
+    id: string
+    networks: string
+    ledger_index: number
+    tx_hash: string
+    date: number
+  }>
+}
+
+/**
+ * Save amendment enabled on a network to the database.
+ *
+ * @param enabledAmendment - The input amendment.
+ *
+ * @returns Void.
+ */
+async function saveAmendmentsEnabled(
+  enabledAmendment: AmendmentEnabled,
+): Promise<void> {
+  await query('amendments_enabled')
```

Comment:
Since `addAmendmentsDataFromJSON` has a `log.info` explaining what the function is doing, should this log as well?

Reply:
`addAmendmentsDataFromJSON` only execute once in a while, while `saveAmendmentsEnabled` will log for each amendments and will swamp the log so I don't think it's necessary here (otherwise we will need to log whenever we add/update one row to the db for other updates as well).

Reply:
Makes sense :)